spec/models/post_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
```

Comment:
- [ ] Your tests are failing. Kindly go through them again and fix the errors.

  ![image](https://github.com/axcell-0/Blog-app/assets/36542986/67bfdc29-6235-4d7c-856d-56dffa3a7409)